Server/src/server/gameCenter/models/game/Game.java

```diff
@@ -558,6 +558,16 @@ private void applyOnAttackSpells(ServerTroop attackerTroop, ServerTroop defender
         }
     }
 
+    private void applyOnCounterAttackSpells(ServerTroop attacker, ServerTroop counterAttacker) {
```

Comment:
Can you move this function under all the lines in which it is used ? This is about clarifying the code, and jumping up and down doesn't help.

Reply:
Done